napari/layers/labels/labels.py

```diff
@@ -341,6 +343,16 @@ def mode(self, mode: Union[str, Mode]):
         self.events.mode(mode=mode)
         self.refresh()
 
+    @property
+    def overwrite(self):
+        return str(self._overwrite)
+
+    @overwrite.setter
+    def overwrite(self, overwrite: Union[str, bool]):
```

Comment:
what does setting with a `str` do vs setting with a `bool`?  It mostly seems like  `bool`  behavior... but the `@property` above always coerces to `str`?  (suggest limiting it to `bool`)